apps/lib/apps.c

```diff
@@ -628,7 +628,7 @@ void app_bail_out(char *fmt, ...)
     exit(1);
 }
 
-void* app_malloc(int sz, const char *what)
+void* app_malloc(size_t sz, const char *what)
```

Comment:
Same as above.

Reply:
Please also note that there is `Could not allocate %d bytes` format string which is clearly incorrect with size_t. Could you change it to `Could not allocate %zu bytes`...?